src/app/app-interactive-map/app-interactive-map.component.ts

```diff
@@ -89,6 +93,25 @@ export class AppInteractiveMapComponent implements OnInit, AfterViewInit {
         this.store.dispatch(new fromActions.Loaded());
         this.loading = false;
       });
+
+    const dialogConfig = new MatDialogConfig();
+    dialogConfig.disableClose = true;
+    dialogConfig.autoFocus = true;
+    dialogConfig.panelClass = 'loader';
+
+    // opening the dialog throws ExpressionChangedAfterItHasBeenCheckedError
+    // See https://github.com/angular/material2/issues/5268#issuecomment-416686390
+    // setTimeout(() => ...., 0);
+    this.store
+      .pipe(
+        select(isLoading),
+      ).subscribe((loading) => {
+        if (loading) {
+          setTimeout(() => this.dialog.open(LoaderComponent, dialogConfig), 0);
```

Comment:
I wanted you to add the comment inside the `if`, just before the `setTimeout` :D https://github.com/DD-DeCaF/caffeine/pull/53/files#diff-fbbc852574ed9718241743f70b6af746R102